examples/tf/erwr_cartpole.py

```diff
@@ -29,7 +29,7 @@ def erwr_cartpole(ctxt=None, seed=1):
     """
     set_seed(seed)
     with LocalTFRunner(snapshot_config=ctxt) as runner:
-        env = GarageEnv(env_name='CartPole-v1')
+        env = GymEnv(env_name='CartPole-v1')
```

Comment:
```suggestion
        env = GymEnv('CartPole-v1')
```

Always call positional arguments in positional form.